frontend/packages/kubevirt-plugin/src/components/vms/vm-console.tsx

```diff
@@ -76,11 +87,28 @@ const VMConsoles: React.FC<VMConsolesProps> = ({
       <VMIsDown />
     );
   }
-
   const vncServiceManual = (vnc && vnc.manual) || undefined;
   const rdpServiceManual = (rdp && rdp.manual) || undefined;
 
-  const desktopViewverSelector = isWindows(vm) ? (
+  const vmName = getName(vm);
+  const namespace = getNamespace(vm);
+  const typeNotSupported =
```

Comment:
let's defer assigning VNC to type in default props, so we will not end up with typeNotSupported here. This way we don't have to show the alert